process-migration-service/frontend/src/component/tabMigrationPlan/MigrationPlansTable.js

```diff
@@ -111,44 +142,34 @@ export default class MigrationPlansTable extends Component {
         cell: {
           formatters: [
             (value, { rowData }) => [
-              <Table.Actions key="0">
+              <Table.Actions key="Actions_Migration">
                 <OverlayTrigger overlay={tooltipExecute} placement={"bottom"}>
                   <Table.Button
                     bsStyle="link"
-                    onClick={() => this.props.openMigrationWizard(rowData)}
+                    onClick={() => this.openMigrationWizard(rowData.id)}
                   >
                     <Icon type="fa" name="play" />
                   </Table.Button>
                 </OverlayTrigger>
               </Table.Actions>,
-              <Table.Actions key="1">
-                <MigrationPlansEditPopup
-                  title="Export Migration Plan"
-                  actionName="Export"
-                  buttonLabel="Copy To Clipboard"
-                  content={JSON.stringify(rowData)}
-                  retrieveAllPlans={this.props.retrieveAllPlans}
-                  updatePlan={this.props.updatePlan}
-                  planId={rowData.id}
-                />
+              <Table.Actions key="Actions_Export">
+                <ExportPlanModal plan={rowData} />
               </Table.Actions>,
-              <Table.Actions key="2">
+              <Table.Actions key="Actions_Delete">
                 <OverlayTrigger overlay={tooltipDelete} placement={"bottom"}>
                   <Table.Button
                     bsStyle="link"
-                    onClick={() => this.props.showDeleteDialog(rowData.id)}
+                    onClick={() => this.showDeleteDialog(rowData.name)}
```

Comment:
the showDeleteDialog expects id, this might be reason, why delete dialog is not being displayed.